src/client/client.ts

```diff
@@ -224,7 +226,8 @@ export default class HTTPClient {
     relativePath: string,
     query?: Query<any>,
     requestHeaders: Record<string, string> = {},
-    jsonOptions: utils.JSONOptions = {}
+    jsonOptions: utils.JSONOptions = {},
+    parseBody: boolean = true
```

Comment:
Do we want to update the JSDoc for this param?

Reply:
Added in e7e928e076a6113574aa8eb53d2e718de96a39df